src/main/common/filter.c

```diff
@@ -68,15 +79,26 @@ FAST_CODE float pt1FilterApply(pt1Filter_t *filter, float input)
 
 // PT2 Low Pass filter
 
+// PTn cutoff correction = 1 / sqrt(2^(1/n) - 1)
+#define CUTOFF_CORRECTION_PT2 1.553773974f
```

Comment:
Put this at the top of the file (line 32), and while you're at it, put the macro for the PT3 cutoff correction there as well

Reply:
done